internal/publish/connect_cloud/content_request_base.go

```diff
@@ -124,10 +124,6 @@ func (c *ServerPublisher) getContentRequestBase(isFirstDeploy bool) (*types.Cont
 		title = c.SaveName
 	}
 
-	rVersion := ""
-	if c.Config.R != nil {
-		rVersion = c.Config.R.Version
-	}
```

Comment:
Do we need to update the configuration spec to specify that that `[r].version` is an invalid field when the `product_type` is Connect Cloud?

Reply:
Ahh thank you for the clarification! I'll take another look with that in mind